src/core_functions/aggregate/distributive/entropy.cpp

```diff
@@ -50,11 +50,12 @@ struct EntropyFunctionBase {
 
 	template <class T, class STATE>
 	static void Finalize(STATE &state, T &target, AggregateFinalizeData &finalize_data) {
-		double count = state.count;
+		double count = static_cast<double>(state.count);
```

Comment:
We have wrappers for these static casts called `NumericCast` in `numeric_utils.hpp`. They will trap conversion errors, so can we switch to using them?